app/controllers/admin/dashboard_controller.rb

```diff
@@ -41,18 +35,38 @@ def prep_projects_grids_for_division_users
 
     @people_grids = {}
     @people.each do |person|
-      @people_grids[person] = initialize_grid(
-        person.active_agent_projects.order(created_at: :desc, updated_at: :desc),
-        include: [:primary_agent, :secondary_agent],
-        per_page: 5,
-        name: "projects_person_#{person.id}",
-        enable_export_to_csv: false
-      )
+      projects = person.active_agent_projects
+      @people_grids[person] = initialize_wice_grid(projects, person, 5)
     end
   end
 
   def prep_logs
     @context = "dashboard"
     @logs = ProjectLog.in_division(selected_division).where(agent_id: @person.id).by_date.page(1).per(10)
   end
+
+  private
+
+  def initialize_wice_grid(projects, person, page)
+    initialize_grid(
+        projects,
+        include: [:primary_agent, :secondary_agent],
+        per_page: page,
+        order: 'projects.order_by_agent',
+        custom_order: {
+            'projects.order_by_agent' => "case when projects.primary_agent_id = #{person.id} and projects.status_value = 'active' and projects.type = 'Loan' then 8
```

Comment:
The logic is good here. How can this be condensed visually, so it doesn't take up so many characters per line. Can the logic be moved into a variable or method? Can we shorten `projects.primary_agent_id` and `projects.secondary_agent_id` in any way?

Also, we try not to use `=>` syntax and instead use JSON style. From the style guide/dev procedures: "Use newer JSON style hash syntax for new and refactored code, even if existing code uses hash rocket style."